### PR TITLE
Add a script for maintaining the version

### PR DESCRIPTION
**Pull request by andrewhalle**
**This commit**
* add the `version` script which bumps the version in `Cargo.toml` and
  modifies the `README.md`

**Why?**
For convenience.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![CI](https://github.com/deepgram-devs/deepgram-rust-sdk/actions/workflows/ci.yaml/badge.svg)](https://github.com/deepgram-devs/deepgram-rust-sdk/actions/workflows/ci.yaml)
 [![crates.io](https://img.shields.io/crates/v/deepgram)](https://crates.io/crates/deepgram)
-[![dowloads](https://img.shields.io/crates/d/deepgram)](https://crates.io/crates/deepgram)
+[![downloads](https://img.shields.io/crates/d/deepgram)](https://crates.io/crates/deepgram)
```
Comment (andrewhalle) [inline README.md:5]: not related, just a typo I saw

File: scripts/version [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env bash
+
+# Usage: version <bump>
+#   bump: [possible values: major, minor, patch, release, rc, beta, alpha]
+
+# Depends
+if ! (command -v cargo-set-version > /dev/null &&
+      command -v jq > /dev/null &&
+      command -V yq > /dev/null)
```
Comment (yottalogical) [inline scripts/version:9]: Is the differing capitalization of `-v` intentional?

Comment (andrewhalle) [reply scripts/version:13]: Not intentional, thanks for the catch! funnily enough, it does the same thing, but I'll still fix

Comment (andrewhalle) [reply scripts/version]: 1f1a0d1

**Review by yottalogical — APPROVED** [review]

**Pull request merged**